.github/workflows/build-caching-maven-samples.yml

```diff
@@ -0,0 +1,29 @@
+name: Build caching Maven sample validation
+
+on: [push]
+
+jobs:
+
+    verify-build-caching-maven-samples:
+        runs-on: ubuntu-latest
+        env:
+            LOG_MSG_EXTENSION_LOADED: "Build operation 'Gradle Enterprise mojo execution' completed"
+            LOG_MSG_EXECUTION_FAILED: "Error executing a GradleEnterpriseListener callback"
```

Comment:
Where are these variables consumed? What impact do they have?

Reply:
just a wrong copy/paste, deleting that